### ISSUE TITLE
Lack of documentation

### ISSUE THREAD

**Issue by JPLeRouzic** [mateigruber/stroked#1]
Hi Matei,
Is it a VR application for stroke rehabilitation?
If so, do you include hemianopsia (half of visual field missing) rehabilitation?
Is it an application for Meta headset or another headset?
Thanks!
Jean-Pierre

**Comment by mateigruber** [comment]
Hey Jean Pierre. Take a look at this video here. It’s a project that I’m
developing for my wife who is a stroke survivor. She also has this problem
with loss of field:

Would you be interested in testing?

https://m.facebook.com/story.php?story_fbid=pfbid02ME15rtYK21yj8wEU5omVcuCbtgFXzvwtWxsJ2ChAWx65YY5ZfSMNaBYLsogHV2X8l&id=1077332767

**Comment by JPLeRouzic** [comment]
> _Would you be interested in testing?_
Yes, my wife had a stroke a year ago, she has hemianopsia, her left visual field has disappeared, but fortunately she has no other sequelae.

But I think that most games in your project are about physical movements and manipulations, isn't?

Anyway we have no idea how to "load" a VR headset with your project. (Could also be a topic in the documentation)

I think that for hemianopsia what is needed is motivating the user to look at the border of the blind visual field .
Literature reports that people may still perceive movements in this area, even if obviously they are blind to static objects.
I wrote a kind of specification for students of my step-brother to (possibly) work on this topic. You can use it as you wish.
[VR project to treat hemianopsia_en.docx](https://github.com/mateigruber/stroked/files/10464899/VR.project.to.treat.hemianopsia_en.docx)

Jean-Pierre
